content/en/docs/ops/deployment/vm-architecture/index.md

```diff
@@ -0,0 +1,129 @@
+---
+title: Virtual Machine Architecture
+description: Describes Istio's high-level architecture for virtual machines.
+weight: 25
+keywords:
+- virtual-machine
+test: n/a
+owner: istio/wg-environments-maintainers
+---
+
+Before reading this document, be sure to review [Istio's architecture](/docs/ops/deployment/architecture/) and [deployment models](/docs/ops/deployment/deployment-models/).
+This page builds on these pages to explain how Istio can be extended to support joining virtual machines into the mesh.
+
+Istio's virtual machine expansion allows connecting workloads outside of a Kubernetes cluster to the mesh.
+This enables legacy applications, or applications not suitable to run in a containerized environment, to get all the benefits that Istio provides to applications running inside Kubernetes.
+
+Many of the features the Kubernetes automatically provides applications are supplemented by Istio.
```

Comment:
Here's my suggested rewording:

For workloads running on Kubernetes, the Kubernetes platform itself provides various functionalities like service discovery, DNS resolution, and health checks which are often missing in virtual machine environments. Istio enables these features for workloads running on virtual machines, and in addition allows these workloads to utillize Istio functionalities like mutual TLS (mTLS), rich telemetry and, advanced traffic management capabilities.